packages/core/src/utils/MessageValidator.ts

```diff
@@ -11,4 +14,26 @@ export class MessageValidator {
   public static validate<T extends object>(classInstance: T) {
     return validateOrReject(classInstance)
```

Comment:
this will still use the `class-validator` error while the `validateSync` method now uses our custom error handling. We should either remove this method, or give it the same care we did in the `validateSync` method

Reply:
I'll remove it